### DISCUSSION TITLE
react text area does not work with input or send keys

### DISCUSSION THREAD

**Discussion by tallevy22** [seleniumbase/SeleniumBase#1082] [Q&A] [answered]
hi ,
i have an issue, i need to type text into a react text area. normal send_keys or input does not work, do you have a suggestion how I can enter a text with sb?
thanks
Tal

**Comment by mdmintz** [comment, marked as answer]
@tallevy22 Is it possible that there’s an iframe on that site and that the text area is inside? If so, you’ll need to enter the iframe first with switch_to_frame(). I would need to see an example to help. Maybe the selector you have chosen is incorrect.

**Reply by tallevy22** [reply]
yes you were correct , the selector I was trying to use was wrong, its not an iframe issue but the wrong selector.
thanks
